benchmarks/_benchmark.py

```diff
@@ -112,14 +112,17 @@ def record(
         )
         return benchmark, output
 
-    def execute_command(self, command):
+    def execute_command(self, command, capture_output=True):
         try:
             print(command)
-            result = subprocess.run(command, capture_output=True, check=True)
+            result = subprocess.run(command, capture_output=capture_output, check=True)
```

Comment:
LGTM

Happy New Year! 🍾

Reply:
@dianaclarke Thank you so much! Happy New Year to you too!